.travis.yml

```diff
@@ -10,6 +10,7 @@ rvm:
   - rbx
 before_install:
   - gem update --system # Need for Ruby 2.5.0. https://github.com/travis-ci/travis-ci/issues/8978
+  - gem install bundler
```

Comment:
NB this fixes issue with running tests on ruby 2.5.1 https://travis-ci.org/Mange/roadie/jobs/330945529